libjob_queue/src/ext_job.c

```diff
@@ -148,6 +147,7 @@ struct ext_job_struct {
 
   bool              private_job;           /* Can the current user/delete this job? (private_job == true) means the user can edit it. */
   bool              __valid;               /* Temporary variable consulted during the bootstrap - when the ext_job is completely initialized this should NOT be consulted anymore. */
+  bool              deprecated;
```

Comment:
Hmmm - smells a bit that we need to internalize this?

Reply:
When using ext_job_json_fprintf, we have to know whether to print from argv or deprecated_argv.
Alternative: 
1) We do a logic check of the population of deprecated_argv and argv.
2) We do not actually need deprecated_argv.